release/golden_notebook_tests/workloads/modin_xgboost_test.py

```diff
@@ -9,19 +9,18 @@
 
 from utils.utils import is_anyscale_connect
 
-HIGGS_S3_URI = "s3://ray-ci-higgs/HIGGS.csv"
-SIMPLE_HIGGS_S3_URI = "s3://ray-ci-higgs/simpleHIGGS.csv"
+FILE_URL = "https://archive.ics.uci.edu/ml/machine-learning-databases/" \
+           "00280/HIGGS.csv.gz"
 
 
 def main():
     print("Loading HIGGS data.")
 
     colnames = ["label"] + ["feature-%02d" % i for i in range(1, 29)]
 
+    data = pd.read_csv(FILE_URL, names=colnames)
```

Comment:
This will download and read the large file always, even in a smoke test. Why can't we continue to use a self-hosted subsampled data source here?

Reply:
There is an S3 permission issue on buildkite that prevents using even public buckets. This is more of a workaround for now, until that can be fixed (will make an issue for it if there isn't one already)

Reply:
Can we use a https link from s3? For example: https://ray-ci-higgs.s3.us-west-2.amazonaws.com/HIGGS.csv

Reply:
OR https://ray-ci-higgs.s3.us-west-2.amazonaws.com/simpleHIGGS.csv

Reply:
Doesn't work either, tried it

Reply:
Can you verify that it's also not possible to download ray wheels (which are also hosted in a ray S3 bucket) on that machine?

Reply:
Explicitly, it'd be good to test that `wget -o test.csv https://ray-ci-higgs.s3.us-west-2.amazonaws.com/simpleHIGGS.csv` doesn't work.